File: News_Service_WebFlux/src/main/java/com/api/AdvancedServiceWebFlux/util/AppConstants.java
package com.api.AdvancedServiceWebFlux.util;

public interface AppConstants {
    String SPACE = " ";

    String NO_SPACE = "";

    String DOT = ".";

    String HEADER_AUTHORIZATION = "Authorization";

    String TOKEN_BEARER = "Bearer" + SPACE;

    String TYPE_APPLICATION_JSON = "application/json";

    String ERRORS = "errorMessage";

    String MESSAGE_STATUS_OK = "Все окей";

    String NO_AUTHORIZATION = "Не авторизованный пользователь";

    String PATTERN_FORMAT_EMAIL = "^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+[A-Za-z]{2,6}$";

    String PATTERN_FORMAT_UUID = "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$";

    String CODES = "codes";
}

File: News_Service_Kotlin/src/main/kotlin/com/dunice/advancedServiceKotlin/util/AppConstants.kt
package com.dunice.advancedServiceKotlin.util

interface AppConstants {
    companion object {
        const val SPACE: String = " "

        const val NO_SPACE: String = ""

        const val DOT: String = "."

        const val HEADER_AUTHORIZATION: String = "Authorization"

        const val TOKEN_BEARER: String = "Bearer" + SPACE

        const val TYPE_APPLICATION_JSON: String = "application/json"

        const val ERRORS: String = "errorMessage"

        const val MESSAGE_STATUS_OK: String = "Все окей"

        const val NO_AUTHORIZATION: String = "Не авторизованный пользователь"

        const val PATTERN_FORMAT_EMAIL: String = "^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+[A-Za-z]{2,6}$"

        const val PATTERN_FORMAT_UUID: String =
            "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$"
    }
}
